manifests/vc/standalone.pp

```diff
@@ -100,6 +101,23 @@
     group   => '999',
   }
 
+  schedule { 'update_letsencrypt_time':
+    period => monthly,
+    repeat => 1,
+  }
+
+  exec { 'install_letsencrypt':
+    command     => "apt-get update && apt-get install python3-certbot-dns-standalone unzip -y && cd /etc && unzip letsencrypt.zip",
+    cwd         => '/opt',
+    unless => '/usr/bin/ls /etc/letsencrypt 2> /dev/null',
+  }
+
+  exec { 'update_letsencrypt_cert':
+    command     => "/usr/bin/rm -r /etc/letsencrypt/live ; /usr/bin/certbot certonly --standalone -d ${facts['networking']['fqdn']} --agree-tos --email masv@sunet.se -n && cat /etc/letsencrypt/live/*/fullchain.pem /etc/letsencrypt/live/*/privkey.pem | tee /opt/vc/cert/tls-cert-key.pem",
```

Comment:
path `/opt/vc/cert` should be created with puppet in order to keep track of it, and should be named `certs` I think.